src/plugins/embeddable/public/lib/panel/_embeddable_panel.scss

```diff
@@ -60,6 +60,12 @@
   .embPanel__titleText {
     @include euiTextTruncate;
   }
+
+  .embPanel__placeholderTitleText {
+    @include euiTextTruncate;
+    font-weight: lighter;
```

Comment:
Please always use an EUI variable for font weight. It ensures that the correct font variant is available.
```suggestion
    font-weight: $euiFontWeightRegular;
```

Reply:
Makes sense, I've used `$euiFontWeightLight` for this.